### ISSUE TITLE
Multiple FAST consequential Voltage Read 

### ISSUE THREAD

**Issue by Saultes45** [Saultes45/VariableFlowGenerator#4]
Where is the try catch statement???

**Comment by Saultes45** [comment]
It is solved with version 1.8. 
The previous versions, <1.7, still suffer from it.

Any movement calls the function: function DisableForMovement(app) 
which in turns executes the following line:
app.ReadButton.Enable                   = 'off';

There is no TRY CATCH statement. It still needs to be implemented BUT it works anyways because the button is disabled.

**TL;DR: use version 1.8**


EDIT: Replaced 1.7 with 1.8 and 1.6 with 1.7 (basically all +0.1)

**Comment by Saultes45** [comment]
a try-catch error has been implemented in the function: sinleManualRead (typo noted by the way)


```
            catch error
                app.NISatusLamp.Color   = 'r';
                Message(1,1,0, 'Could not perform the voltage read function', 'KO', app.TestIDEditField.Value);                
                uialert(app.UIFigure,'Could not perform the voltage read function','Communication error');
            end
```
